packages/cmd/src/help/abstract/AbstractHelpGenerator.ts

```diff
@@ -173,7 +173,7 @@ export abstract class AbstractHelpGenerator implements IHelpGenerator {
 
             // build the option help text
             let optionText = option.description;
-            const defaultValueText = option.defaultValue ? this.grey("\nDefault value: " + option.defaultValue) : "";
+            const defaultValueText = option.defaultValue !== undefined ? this.grey("\nDefault value: " + option.defaultValue) : "";
```

Comment:
Thanks for fixing this for cases where `defaultValue = false`.

The strict check for undefined means that `defaultValue = null` would be displayed in help content. Since `null` and `undefined` should both be invalid for `defaultValue`, could you change the check to:
```typescript
option.defaultValue != null
```

Reply:
Hi @tjohnsonBCM ,
now I have a new commit. It checks both null and undefined.